README.md

```diff
@@ -21,6 +21,7 @@ Getting Started
 1. You will require a GoG or any older version of Might and Magic 7 installed for engine to guess game assets folder. You can override assets/game path in launcher or by using WOMM_PATH_OVERRIDE env variable.
 2. Use any IDE with CMake integration.
 3. Build & run game directly, or via launcher.
+4. Please note some additional files need to be copied from the Git repo 'resources' folder.
```

Comment:
2. Copy WoMM-specific resources from the 'resources' folder of the repo to the game assets folder.
3. Use any IDE with CMake integration.
4. Build & run the game directly, or via launcher.